pkgs/development/libraries/noise-suppression-for-voice/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "noise-supression-for-voice";
+  version = "2020-10-12";
```

Comment:
According to: https://nixos.org/manual/nixpkgs/stable/#chap-reviewing-contributions :

>  If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 

So `pname` should have `-unstable` at the end.